swift-tools-support-core/Sources/TSCBasic/Path.swift

```diff
@@ -92,6 +92,18 @@ public struct AbsolutePath: Hashable {
         try self.init(PathImpl(validatingAbsolutePath: path))
     }
 
+    #if compiler(>=5.3)
```

Comment:
Can you add this as an extension to TSCTestSupport target? I don't really want to expose this hack to non-test code. Unfortunately, things are not completely setup completely with TSC so I'll appreciate if you can also cherry-pick the TSC change into the https://github.com/apple/swift-tools-support-core repo 🙇‍♂️